ward/terminal.py

```diff
@@ -370,7 +487,45 @@ def output_all_test_results(
         output_tests = self.runtime_output_strategies.get(
             self.test_output_style, output_test_per_line
         )
-        all_results = output_tests(fail_limit, test_results_gen)
+
+        spinner = SpinnerColumn(style="pass.textonly")
```

Comment:
This should only be done if `TestProgressStyle.BAR in progress_styles and num_tests > 0`